src/krishna/BankingSystem1.java

```diff
@@ -0,0 +1,82 @@
+package krishna;
+//Create a Banking System which has the following functionality.
+
+public class BankingSystem1 {
+	String name;
+	int totalbalance;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.